server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -1713,9 +1717,6 @@
             <param name="fiatCurrency" />
             <param name="coinmarketcapApiKey"/>
             <cryptocurrency>LSK</cryptocurrency>
-            <cryptocurrency>BTC</cryptocurrency>
-            <cryptocurrency>LTC</cryptocurrency>
-            <cryptocurrency>ETH</cryptocurrency>
```

Comment:
there is another binance rate source implementation for these cryptos that doesn't need coinmarketcapApiKey